### ISSUE TITLE
Improve logging with library

### ISSUE THREAD

**Issue by Electrenator** [Electrenator/Tabbly#29]
Recently seen that logging libraries could be really nice to have for error and other logging. Should allow the output of Tably to at least have a timestamp and maybe also have logging libraries. Should still look at what's available for Python.

Don't think it's neassecary for it to log to a file by default. Except maybe error logging with an auto start (#27).

**Comment by Electrenator** [comment]
Want to have options for logging setting with it. Should be able to do that after #30, which I am currently reading some articles for

**Comment by Electrenator** [comment]
Oh, well apparently python has it's own [loging lib](https://docs.python.org/3/library/logging.html)
